subprojects/file-watching/src/integTest/groovy/org/gradle/internal/watch/BuildOperationsFileSystemWatchingIntegrationTest.groovy

```diff
@@ -16,10 +16,12 @@
 
 package org.gradle.internal.watch
 
+import com.gradle.enterprise.testing.annotations.LocalOnly
 import org.gradle.integtests.fixtures.BuildOperationsFixture
 import org.gradle.internal.watch.vfs.BuildFinishedFileSystemWatchingBuildOperationType
 import org.gradle.internal.watch.vfs.BuildStartedFileSystemWatchingBuildOperationType
 
+@LocalOnly
```

Comment:
In order for the annotation to take effect, we'd also need to configure test selection in a way it respects the annotation, right? Like described here: https://docs.gradle.com/enterprise/test-distribution/#gradle_executor_restrictions_annotation_matcher

Reply:
Correct, added